tests/testExpressionFactor.cpp

```diff
@@ -434,17 +430,17 @@ TEST(ExpressionFactor, composeTernary) {
   // Check unwhitenedError
   std::vector<Matrix> H(3);
   Vector actual = f.unwhitenedError(values, H);
-  EXPECT_LONGS_EQUAL(3, H.size());
-  EXPECT(assert_equal(I_3x3, H[0],1e-9));
-  EXPECT(assert_equal(I_3x3, H[1],1e-9));
-  EXPECT(assert_equal(I_3x3, H[2],1e-9));
+  EXPECT_LONGS_EQUAL(3, H.size())
+  EXPECT(assert_equal(I_3x3, H[0],1e-9))
```

Comment:
Formatting: missing a space after the comma

Reply:
Thanks ! Note I won't change as orthogonal to the PR - there are *many* formatting issues with al tests.